docs/red_eyes_mainnet_tutorial.rst

```diff
@@ -0,0 +1,142 @@
+        "balance": 2000,
+        "settle_timeout": 600
+    }
+TODO: Update to have correct values
+
+At this point the specific value of the ``balance`` field isn't too important, since it's always possible to :ref:`deposit more tokens <topping-up-a-channel>` to a channel if need be.
+
+Successfully opening a channel returns the following information::
+
+    {
+        "channel_identifier": "0x2a65aca4d5fc5b5c859090a6c34d164135398226",
+        "partner_address": "0x61c808d82a3ac53231750dadc13c777b59310bd9",
+        "balance": 2000,
+        "state": "opened",
+        "settle_timeout": 60
+    }
+
+TODO: check correctness
+
+.. _doing-payments:
+
+Payments
+========
+Now that we have joined a token network, we can start making payments. For this, we need the address of the RDN token and the address of the recipient of the payment::
+
+    POST /api/v2/transfers/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6/0x61c808d82a3ac53231750dadc13c777b59310bd9
+
+In the payload we specify the amount of RDN tokens of the payment::
+
+    {
+        "amount": 42
+    }
+TODO: Check for correctness. What denomination are we using for amount?
+
+In this specific case the payment goes directly to one of our channel partners, since we have an open channel with ``0x61c808d82a3ac53231750dadc13c777b59310bd9``. If we specify an address that we do not have a direct channel with, the Raiden Network finds a path to the target address and use mediated transfers to make a payment from our address to the target address.
+
+It's as simple as that to do payments using the Raiden Network.
+
+Let's say we know someone with the address ``0x00014853D700AE1F39BA9dbAbdeC1c8683CF1b2A``, who is also part of the RDN token network. Even though we do not have a channel with this person it is as easy as above to send a payment. All we need is the address::
+
+    POST /api/v2/transfers/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6/0x00014853D700AE1F39BA9dbAbdeC1c8683CF1b2A
+
+With the payload of the amount of RDN tokens we want to pay::
+
+    {
+        "amount": 73
+    }
+TODO: Check for correctness.
+
+Just like this we can send payments to anyone who is part of the token network for the RDN token.
+
+.. _topping-up-a-channel:
+
+Depositing tokens
+=================
+If we are spending more tokens than we are receiving and hence depleting our channels, it it possible to "top up" channels. For this we need the channel identifier of the channel we want to top up::
+
+    PATCH /api/v2/channels/0x2a65aca4d5fc5b5c859090a6c34d164135398226
+
+with the payload::
+
+    {
+        "total_deposit": 4000
+    }
+TODO: Check for correctness
+
+Here it is important to note that the ``total_deposit`` value is idempotent. This means that we do not specify the amount to add to our initial deposit, but the total amount. In this specific case, since we initially deposited 2000 RDN tokens, and now provide a ``total_deposit`` of 4000, we are adding 2000 more RDN tokens to our channel.
```

Comment:
The vaule is not idempotent, the operation is. Also not every reader will know what idempotence is and why it matters. I suggest to rewrite this to something like: "Notice that we need to specify the total deposit, not the amount we wish to top up: We initially deposited 2000 tokens and want to add 2000 more, so we give a total_deposit of 4000. This way the topup request is idempotent - if it is sent to us repeatedly (by accident or as part of an attack) it will have no further effect."